src/themes/journey/theme.json

```diff
@@ -949,6 +949,35 @@
     }
   },
 
+  "richTextBlock": {
+    "base": {
+      "*:first-child": { "mt": 0 }
+    },
+    "panel-primary": {
+      "py": "lg",
+      "px": ["sm", "52px"],
```

Comment:
please avoid magic numbers, (52px), can we make sure were using numbers from theme and speak to designers if they use a value that's not in the theme tokens

Reply:
Spoke with design and updated to remove magic numbers